README.md

```diff
@@ -1 +1,83 @@
-# Vahini-2.0
+# Welcome to Project Vahini
+
+Project Vahini is a project that is used to automatically move a watering vehicle on the farmland or argricultural plantations and water the trees.
+
+## About
+@amartyanambiar @healertrix write something here
+
+## Setup Instructions
+
+### Pre-requistes for setup
+- Python 3.7 or higher
+- An idea on how to use a terminal
+- Git Installed on your computer
+
+### Setup
+
+1. Clone the repository using the below command
+
+```
+git clone https://github.com/Blake2912/Vahini-2.0.git
+```
+
+2. Setup Virtual Environment
```

Comment:
I don't believe we need to be soo very precise and derailed about the environment.
Let's just post - Recommend to be run in a virtual environment. And mention a medium article or something on how to create one.

Reply:
Let it be if it's detailed it will be easy for us for faster setup and reference

Reply:
Okay